google-map.html

```diff
@@ -511,217 +147,295 @@
     }
 
   </style>
+  <template>
+    <google-maps-api 
+      api-key="{{apiKey}}" 
+      client-id="{{clientId}}" 
+      version="{{version}}" 
+      libraries="{{libraries}}" 
+      signed-in="{{signedIn}}" 
+      language="{{language}}"
+      on-api-load="mapApiLoaded"></google-maps-api>
+
+    <div id="map"></div>
+    
+    <content id="markers" select="google-map-marker"></content>
+
+  </template>
+</dom-module>
 
-  <google-maps-api apiKey="{{apiKey}}" clientId="{{clientId}}" version="{{version}}" on-api-load="{{mapApiLoaded}}" libraries="{{libraries}}" signedIn="{{signedIn}}" language="{{language}}"></google-maps-api>
-
-  <div id="map"></div>
-
-  <content id="markers" select="google-map-marker"></content>
-
-</template>
 <script>
 
   Polymer({
 
-    /**
-     * A Maps API key. To obtain an API key, see developers.google.com/maps/documentation/javascript/tutorial#api_key.
-     *
-     * @property apiKey
-     * @type string
-     */
-    apiKey: null,
+    is: 'google-map',
 
-    /**
-     * A Maps API for Business Client ID. To obtain a Maps API for Business Client ID, see developers.google.com/maps/documentation/business/.
-     * If set, a Client ID will take precedence over an API Key.
-     *
-     * @attribute clientId
-     * @type string
-     */
-    clientId: null,
+    properties: {
+      /**
+       * A Maps API key. To obtain an API key, see developers.google.com/maps/documentation/javascript/tutorial#api_key.
+       *
+       * @property apiKey
+       * @type string
+       */
+      apiKey: String,
 
-    /**
-     * A latitude to center the map on.
-     *
-     * @attribute latitude
-     * @type number
-     * @default 37.77493
-     */
-    latitude: 37.77493,
+      /**
+       * A Maps API for Business Client ID. To obtain a Maps API for Business Client ID, see developers.google.com/maps/documentation/business/.
+       * If set, a Client ID will take precedence over an API Key.
+       *
+       * @attribute clientId
+       * @type string
+       */
+      clientId: String,
 
-    /**
-     * A comma separated list (e.g. "places,geometry") of libraries to load
-     * with this map. Defaults to "places". For more information see
-     * https://developers.google.com/maps/documentation/javascript/libraries.
-     *
-     * @attribute libraries
-     * @type string
-     * @default "places"
-     */
-    libraries: "places",
+      /**
+       * A latitude to center the map on.
+       *
+       * @attribute latitude
+       * @type number
+       * @default 37.77493
+       */
+      latitude: {
+        type: Number,
+        value: 37.77493
```

Comment:
lat/lng should probably notify. I'd imagine people what to be notified on those. Also probably `relflectToAttribute`.